Add Kotlin conformance test runner (#809)
partiql/partiql-lang-kotlin: test/partiql-tests-runner/README.md

# PartiQL Kotlin Test Runner

This package checks whether the conformance tests (in [partiql-tests](https://github.com/partiql/partiql-tests)) run 
using the `partiql-lang-kotlin` implementation return the correct result.

> This package will allow us to:
> 1. verify conformance tests are defined correctly (e.g. verify evaluation environment is not missing a table)
> 2. identify areas in the Kotlin implementation that diverge from the PartiQL specification

Reviewer: ```suggestion
This package enables:
1. Verifying conformance tests are defined correctly (e.g. verify evaluation environment is not missing a table)
2. Identifying areas in the Kotlin implementation that diverge from the [PartiQL Specification](https://partiql.org/assets/PartiQL-Specification.pdf)
```

> Eventually, the Kotlin test runner should replace the `pts` and `testscript` Gradle subprojects along with some other 

Reviewer: ```suggestion
Eventually, the `partiql-test-runner` module will replace the `partiql-pts` and `partiql-testscript` modules along with some other 
```

Author: Thanks for the catch
